app/Providers/ImporterServiceProvider.php

```diff
@@ -56,28 +56,15 @@ protected function registerRoutes()
      */
     protected function registerConfig()
     {
-        $this->mergeConfigFrom(__DIR__.'/../Config/config.php', 'importer');
```

Comment:
These configs don't need this, I think just create files like `config/installer.php` or `config/importer.php` and move the configs there. The `mergeConfigFrom` is only needed if it's a module

Reply:
Oh, okay